### PR TITLE
MSLV-108-2

### PR DESCRIPTION
**Pull request by joshfoss4**

### CODE REVIEW

File: src/index.test.ts [hunk 1]
```diff
@@ -1,13 +1,62 @@
...
+    port: 9999,
+    ip: "127.0.0.1",
+    protocol: Protocol.tcp
 });
 
-describe("Test Scoreboard Listener", () => {
-  test("Scoreboard should be listening", async () => {
-    expect((await scoreboard).isListening).toEqual(true);
-  });
+describe("Expert Mode Listener", () => {
```
Comment (karlsnyder0) [inline src/index.test.ts:11]:
```suggestion
let scoreboard;

describe("Expert Mode Listener", () => {
    before(async () => {
        scoreboard = await LynxScoreboard.listen({
            port: 9999,
            ip: "127.0.0.1",
            protocol: Protocol.tcp
        });
    });

    test("Expert Mode should be listening", () => {
        expect(scoreboard.isListening).toEqual(true);
    });

    test("Expert Mode should stop listening", async () => {
        await scoreboard.stopListening();

        expect(scoreboard.isListening).toEqual(false);
    });
});
```

File: src/index.test.ts [hunk 2]
```diff
@@ -1,13 +1,62 @@
 import "jest";
-import { LynxScoreboard } from "./lynx-scoreboard";
+import { LynxScoreboard, Protocol } from "./lynx-scoreboard";
 
+// Expert mode test
 const scoreboard = LynxScoreboard.listen({
-  port: 9999,
-  ip: "127.0.0.1",
+    port: 9999,
```
Comment (karlsnyder0) [inline src/index.test.ts:6]:
```suggestion
```

File: src/lynx-scoreboard.ts [hunk 3]
```diff
@@ -120,6 +131,15 @@ export class LynxScoreboard {
                 this._publish("error", err.message);
             })
             .bind(this.port, this.ip);
+    }
+
+    private startListeningTCP(): void {
+        // Start TCP listener
+        this._serverTCP.listen(this.port, this.ip);
```
Comment (karlsnyder0) [inline src/lynx-scoreboard.ts:138]: Use- https://nodejs.org/docs/latest-v14.x/api/net.html#net_net_createserver_options_connectionlistener

File: src/index.test.ts [hunk 4]
```diff
@@ -1,13 +1,62 @@
 import "jest";
-import { LynxScoreboard } from "./lynx-scoreboard";
+import { LynxScoreboard, Protocol } from "./lynx-scoreboard";
 
+// Expert mode test
```
[the remark this answers is not in the document]
Comment (joshfoss4) [reply src/index.test.ts]: Much better!

File: src/lynx-scoreboard.ts (hunk 3 above)
Comment (joshfoss4) [reply src/lynx-scoreboard.ts]: I think everything was here from the docs, but the comments were off and the order was a bit confusing.  Is this better?

**Review by karlsnyder0 — APPROVED** [review]
Soft approval.  Minor tweaks as discussed.  Looks great!

**Review by JEltgroth — APPROVED** [review]

**Pull request merged**